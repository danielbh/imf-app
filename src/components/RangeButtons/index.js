import React from 'react';
import RangeButton from '../../containers/RangeButton';
import FlexRowEnd from '../../components/FlexRowEnd';
import {
  WEEK,
  MONTH,
  THREE_MONTHS,
  YEAR,
  ALL
} from "../../containers/App/constants";

const RangeButtons = () => (
  <FlexRowEnd>
    <RangeButton dateRange={WEEK}>1W</RangeButton>
    <RangeButton dateRange={MONTH}>1M</RangeButton>
    <RangeButton dateRange={THREE_MONTHS}>3M</RangeButton>
    <RangeButton dateRange={YEAR}>1Y</RangeButton>
    <RangeButton dateRange={ALL}>ALL</RangeButton>
  </FlexRowEnd>
)

export default RangeButtons;