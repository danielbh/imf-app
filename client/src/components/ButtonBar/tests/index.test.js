import React from 'react';
import { shallow } from 'enzyme';
import RangeButtons from '../index'

describe('<ButtonBar />', () => {
  it('matches snapshot', () => {
     expect(shallow(<RangeButtons />)).toMatchSnapshot();
  });
});