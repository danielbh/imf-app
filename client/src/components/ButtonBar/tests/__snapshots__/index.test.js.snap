// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonBar /> matches snapshot 1`] = `
<Flexbox>
  <Connect(Link)
    dateRange="WEEK"
  >
    1W
  </Connect(Link)>
  <Connect(Link)
    dateRange="MONTH"
  >
    1M
  </Connect(Link)>
  <Connect(Link)
    dateRange="THREE_MONTHS"
  >
    3M
  </Connect(Link)>
  <Connect(Link)
    dateRange="YEAR"
  >
    1Y
  </Connect(Link)>
  <Connect(Link)
    dateRange="ALL"
  >
    ALL
  </Connect(Link)>
</Flexbox>
`;
